roles/openshift_logging/defaults/main.yml

```diff
@@ -0,0 +1,64 @@
+---
+image_prefix: docker.io/openshift/origin-
+image_version: latest
+logging_use_ops: False
+master_url: https://kubernetes.default.svc.cluster.local
```

Comment:
This can become `"https://kubernetes.default.svc.{{ openshift.common.dns_domain }}"`. We currently hardcode `dns_domain` as `cluster.local`, but that isn't guaranteed into the future.